packs/aws/README.md

```diff
@@ -118,6 +118,7 @@ following values in datastore:
 - aws.aws_access_key_id
 - aws.aws_secret_access_key
 - aws.region
+- aws.max_number_of_messages (must be between 1 - 10)
```

Comment:
Would also be good to document what this attribute controls / means - aka how many messages to fetch in a batch or some similar explanation.